modules/create-vm/cmd/create-vm/main.go

```diff
@@ -51,10 +49,10 @@ func main() {
 	if err := vmCreator.OwnVolumes(vm); err != nil {
 		exit.ExitFromError(OwnVolumesErrorExitCode, err)
 	}
-	runStrategy := cliOptions.GetRunStrategy()
-	if cliOptions.GetStartVMFlag() && kubevirtv1.RunStrategyAlways != kubevirtv1.VirtualMachineRunStrategy(runStrategy) {
-		err := vmCreator.StartVM(vm.Namespace, vm.Name)
-		if err != nil {
+
+	if cliOptions.GetStartVMFlag() &&
+		(vm.Spec.RunStrategy == nil || *vm.Spec.RunStrategy != kubevirtv1.RunStrategyAlways) {
```

Comment:
What happens if `spec.running` is already `true`, i.e. the VM is already running?

Reply:
It caused an error and should be fixed now.